### DISCUSSION TITLE
Security Onion 16.4 - Kibana/Elastic Fails to Init / Service Unavailable

### DISCUSSION THREAD

**Discussion by postrow** [Security-Onion-Solutions/securityonion#11505] [Unsupported Versions]
For some unexplained reason, Kibana (SO v16.4) fails to launch on a fresh new install. This is the log from the session where we can't get Kibana/Elastalert up and running in Security Onion v16.4.  We reinstalled SO and unfortunately, the issue was not resolved. I was hoping someone could provide me with guidance/suggestions/tricks that may help fix the issue.  THANKS!

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/147430683/12bd570d-776a-465a-b3c6-d1b8a3cd0a7a)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/147430683/8e565b9a-3653-4ea0-9246-8bda853e6f2d)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/147430683/e9914f92-1f5e-4caf-8c9c-e3a0b4ca7c46)

**Comment by cm-ops** [comment]
https://blog.securityonion.net/2021/04/security-onion-1604-has-reached-end-of.html 16.04 has been EOL for a couple years now. Recommend upgrading to 2.4.
